bin/local.js

```diff
@@ -68,7 +69,10 @@ function onFileChange(evt, filepath) {
 }
 
 function startFileWatch() {
-    watcher.watch(config.sourceLocation, {
+    if (fswatcher){
+        fswatcher.close();
```

Comment:
I think that depends if fswatcher.close() sets `fswatcher` to falsy. Assuming it stays an object, it would not be falsy. 
If it isn't falsy, you could end up with a state where no file watcher is active but calls to startFileWatch would not create a new file watcher. 
A cursory look through the chokidar docs didn't reveal any properties of a watcher object that we could read to determine if it is active or inactive.